operator/src/main/resources/scripts/startNodeManager.sh

```diff
@@ -286,7 +286,29 @@ export VM_TYPE="HotSpot"
 #  Copied from ${DOMAIN_HOME}/bin/startNodeManager.sh 
 export NODEMGR_HOME="${NODEMGR_HOME?}"
 export DOMAIN_HOME="${DOMAIN_HOME?}"
-export JAVA_OPTIONS="${JAVA_OPTIONS} -Dweblogic.RootDirectory=${DOMAIN_HOME}"
```

Comment:
Please update the comment at the top of this file to describe NODEMGR_MEM_ARGS and
NODEMGR_JAVA_OPTIONS (line 34 - at the end of the list of optional env vars).